tap_sftp/singer_encodings/csv_handler.py

```diff
@@ -2,6 +2,8 @@
 import csv
 import io
 import os
+import singer
```

Comment:
@radbrt I dont see any references to this new import, is it needed?

Reply:
True, I have no clue how that got there. Removing.